classic-editor.php

```diff
@@ -1,30 +1,63 @@
-<?php
+<?php 
 /**
  * Classic Editor
  *
  * Plugin Name: Classic Editor
- * Plugin URI:  https://wordpress.org
- * Description: Enables the WordPress classic editor and the old-style Edit Post screen layout (TinyMCE, meta boxes, etc.). Supports the older plugins that extend this screen.
- * Version:     1.0-beta
+ * Plugin URI:  https://wordpress.org/plugins/classic-editor/
+ * Description: Enables the WordPress Classic Editor the old-style edit screen layout (TinyMCE, Meta Boxes, etc...) and support the add-ons that extend it.
+ * Version:     1.0.0-beta
  * Author:      WordPress Contributors
- * License:     GPL-2.0+
+ * Author URI:  https://github.com/WordPress/classic-editor/
+ * License:     GPLv2 or later
  * License URI: http://www.gnu.org/licenses/old-licenses/gpl-2.0.html
  * Text Domain: classic-editor
  * Domain Path: /languages
- *
+ * Network:     true
  *
  * This program is free software; you can redistribute it and/or modify it under the terms of the GNU
- * General Public License version 2, as published by the Free Software Foundation.  You may NOT assume
+ * General Public License version 2, as published by the Free Software Foundation. You may NOT assume
  * that you can use any other version of the GPL.
  *
  * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
  * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
  *
+ * Tweaked Build 2018-12-02 based on latest GitHub code -- see all # TWEAK (tag description) for the changes
+ *
+ * Tweaks description:
+ *
+ * - Concise plugin description (detailed plugin guidelines not allow more of 151 characters)
+ * - Use of use Semantic Versioning MAJOR.MINOR.PATCH 1.0.0-beta instead 1.0-beta
+ * - Corrected license spacing
+ * - Network support (multisite)
+ * - Secureds ABSPATH code
+ * - Corrected capitalization for Block Editor and Classic Editor
+ * - Removes redundant words to maintain description in the same line
+ * - Corrects the case where the plugin updates lose and delete user-defined custom settings
+ * - Corrects the case where the plugin deactivation lose and delete user-defined custom settings
+ *
  */
 
-if ( ! defined( 'ABSPATH' ) ) {
-	die( 'Invalid request.' );
-}
+defined( 'ABSPATH' ) || die( 'Invalid request.' ); # TWEAK: alternative code more secure?

```

Comment:
Nope, not more secure. It's exactly the same just written in a bit more "obscure" way :)